pkg/odo/genericclioptions/context.go

```diff
@@ -563,3 +465,72 @@ func ApplyIgnore(ignores *[]string, sourcePath string) (err error) {
 	}
 	return nil
 }
+
+// checkIfConfigurationNeeded checks against a set of commands that do *NOT* need configuration.
+func checkIfConfigurationNeeded(command *cobra.Command) (bool, error) {
```

Comment:
This should be refactored to use a switch instead of many ifs…

Reply:
I know :( since we use other and / or statements in the `if`'s we'll have to stick with ifs instead of case (for now).

Reply:
@cdrage Can we use a if/ else-if ladder?